CONTRIBUTING.md

```diff
@@ -3,31 +3,28 @@
 This is Horusec contributing guide. Please read the following sections to learn how to ask questions and how to work on something.
 
 ## **Table of contents**
-1. [**Getting Started**](#Getting-started)
-   1. [**Before you contribute**](#Before-you-contribute)
-   2. [**Code of Conduct**](#Code-of-Conduct)
-   3. [**Legal**](#Legal)
-2. [**Prerequisites**](#Prerequisites)
-   1. [**Developer Certificate of Origin**](#Developer-Certificate-of-Origin)
-   2. [**Code Review**](#Code-Review)
-   3. [**Pull Requests**](#Pull-Requests)    
-3. [**How to contribute?**](#How-to-contribute?)
-      1. [**Prepare your development environment**](#Prepare-your-development-environment)
-      2. [**First contribution**](#First-contribution)
-      4. [**Add new feature, bugfixing or improvement**](#Add-new-feature-bugfixing-or-improvement)
-      5. [**Pull Request's approval**](#Pull-Request's-approval)
-      6. [**After your pull request's approval**](#After-your-pull-request's-approval)
-4. [**Community**](#Community)
-
-## **Getting started**
-
-### **Before you contribute**
+### 1. [**Before you contribute**](#before-you-contribute)
+> #### i.  [**Code of Conduct**](#code-of-conduct)
+> #### ii. [**Legal**](#legal)
+### 2. [**Prerequisites**](#prerequisites)
+> #### i.   [**Developer Certificate of Origin**](#developer-certificate-of-origin)
+> #### ii.  [**Code Review**](#code-review)
+> #### iii. [**Pull Requests**](#pull-requests)    
+### 3. [**How to contribute?**](#how-to-contribute?)
+ > #### i. [**Prepare your development environment**](#prepare-your-development-environment)
+> #### ii.  [**First contribution**](#first-contribution)
+> #### iii. [**Add new feature, bugfixing or improvement**](#add-new-feature-bugfixing-or-improvement)
+> #### iv.  [**Pull Request's approval**](#pull-request-approval)
+> #### v.   [**After your pull request's approval**](#after-your-pull-request-approval)
+### 4. [**Community**](#community)
```

Comment:
```suggestion
### 1. [**Before you contribute**](#before-you-contribute)
> #### 1.1.  [**Code of Conduct**](#code-of-conduct)
> #### 1.2. [**Legal**](#legal)
### 2. [**Prerequisites**](#prerequisites)
> #### 2.1.   [**Developer Certificate of Origin**](#developer-certificate-of-origin)
> #### 2.2.  [**Code Review**](#code-review)
> #### 2.3. [**Pull Requests**](#pull-requests)    
### 3. [**How to contribute?**](#how-to-contribute?)
> #### 3.1. [**Prepare your development environment**](#prepare-your-development-environment)
> #### 3.2.  [**First contribution**](#first-contribution)
> #### 3.3. [**Add new feature, bugfixing or improvement**](#add-new-feature-bugfixing-or-improvement)
> #### 3.4.  [**Pull Request's approval**](#pull-request-approval)
> #### 3.5.   [**After your pull request's approval**](#after-your-pull-request-approval)
### 4. [**Community**](#community)
```

Roman numbers can get confusing here wouldn't it be better to stick to the standard numbering?